broker/broker-undelivered-messages-ui/src/main/webapp/index.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import React from "react"
+import {render} from "react-dom"
+import store from './store'
+import {
+    isAllSelected,
+    displayNumSelected,
+    getSelectedIds,
+    getSelectedMessages,
+    getIds
+} from './reducer'
+import {
+    addMessage,
+    checkAllMessages,
+    togglePolling,
+    expandAllMessages,
+    updateDeleted,
+    removeMessages
+} from './actions'
+import TableView from './table.view'
+
+require('es6-promise').polyfill()
+require('isomorphic-fetch')
+
+const getMessagesURI = '/admin/jolokia/exec/mil.af.gdes.message.broker.UndeliveredMessages:service=UndeliveredMessages/getMessages/jms.queue.DLQ/Core/'
+
+const Json = ({value}) => (
+    <pre>{JSON.stringify(value, null, 2)}</pre>
+)
+
+const retrieveData = (data, url) => {
+    return window.fetch(url, {credentials: 'same-origin'})
+        .then((res) => res.json())
+        .then((json) => {
+                json.value.filter((message) => !(getIds(data).includes(message.messageID))).map((message) => {
```

Comment:
I think `includes` is an ES6 array method which may cause issues in older browsers. We could polyfill or use `lodash.includes`.